packages/system/src/theme.ts

```diff
@@ -28,30 +28,43 @@ export type ScaleValue<T> = T | T[] | NestedScaleDict<T> | undefined;
  * descriptive name for the scale (e.g. `small`/`medium`/.. or `body`/`heading`/...),
  * and the value is the CSS value.
  */
-export type Scale<T> = {
+export interface Scale<T> {
   [key: string]: ScaleValue<T>;
-};
+}
+
+/**
+ * A {@link Scale} that also includes a required `none` value, which is
+ * usually used to define the blank value (e.g `0`).
+ */
+export interface ZeroScale<T> extends Scale<T> {
+  none: ScaleValue<T>;
+}
 
 /**
  * Predefined {@link Scale} scale which uses size values.
  */
-export type SizeScale<T> = {
+export interface SizeScale<T> {
+  regular?: ScaleValue<T>;
+
```

Comment:
Is this line intentionally?

Reply:
yes, we didn't have enough values and didn't hat the "base" value.

Reply:
Doing this PR I actually find our scale cumbersome ... open props originally used numbers which might be the better approach

Reply:
I mean the blank line :) its also in other files, a blank line after "regular" or "none"

Reply:
I did it to make the value stand out :D

Reply:
ok for me 😀